internal/testhelpers/make.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 const (
-	testTaskQueue = "hello_world"
```

Comment:
removed so that we stop hard-coding this and instead use the test case name as the task queue for each test case.